e2e/hive/clients/polard/Dockerfile

```diff
@@ -14,21 +14,17 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # Use the latest foundry image
-FROM polard-base:test-hive as polaris-hive
+FROM polard/base:v0.0.0 as polaris-hive
```

Comment:
probably want to make this :latest or some dev thing or something

Reply:
does this mean make it polard-base:latest, or pass in as a build arg? the build arg option causes some problems since we don't have access to the hive build command so we can't set it ourselves.

the thinking behind making is polard/base:v0.0.0 was to reuse the same base image as the e2e localnet. should localnet use a specific for testing base image tag instead as well then? i.e like polard/base:testing